CONTRIBUTING.rst

```diff
@@ -28,14 +28,14 @@ Co-develop Zenodo
 -----------------
 Are you missing a feature in Zenodo? Why not help us built the feature. We are
 happy to receive both tiny and large contributions to Zenodo. Have a look at
-our Development Process and Project Life Cycle and don't hesitate to get in
+our `Development Process <https://github.com/zenodo/zenodo/blob/master/docs/development/process.rst>`_ and `Project Life Cycle <https://github.com/zenodo/zenodo/blob/master/docs/development/projectlifecycle.rst>`_ and don't hesitate to get in
```

Comment:
Since this is documentation, perhaps it would be better if those two links resolved to the compiled docs, and not the GitHub source files? Moverover, since we might move from readthedocs to another service at some point use relative link to a "label" instead of readthedocs URL, see: https://stackoverflow.com/questions/37553750/how-can-i-link-reference-another-rest-file-in-the-documentation

Reply:
Almost :-)